d3a_interface/simulation_config/__init__.py

```diff
@@ -0,0 +1,11 @@
+
+
+class SimulationConfigException(Exception):
+    pass
+
+
+def validate_simulation_config(simulation_config):
```

Comment:
In order to know which simulation_config is meant here in the future, please pre- or append something like `api_client` to the name. Also to the exception and the package name. Thanks!